src/lightning/pytorch/callbacks/model_checkpoint.py

```diff
@@ -129,6 +129,8 @@ class ModelCheckpoint(Checkpoint):
             where both values for ``every_n_epochs`` and ``check_val_every_n_epoch`` evenly divide E.
         save_on_train_epoch_end: Whether to run checkpointing at the end of the training epoch.
             If this is ``False``, then the check runs at the end of the validation.
+        override_existing: Whether to overwrite the checkpoint file.
```

Comment:
Did you mean overwrite? Also, the expression "overwrite existing" is probably redundant, since overwriting can only be done if something exists, otherwise it would just be called "writing".

Reply:
I've followed the suggested option in the discussion. Do you find `overwrite_existing` more suitable?
The purpose of this flag is exactly to reuse passed filepath if such exists. This is mainly useful when you have some pipeline run on top of the training and you need your checkpoint to end up in some specific location.

Reply:
```suggestion
        overwrite_existing: Whether to overwrite the checkpoint file.
```